.travis.yml

```diff
@@ -9,6 +9,5 @@ matrix:
   include:
   - jdk: oraclejdk8
   - jdk: oraclejdk9
-  allow_failures:
-  - jdk: oraclejdk9
+  - jdk: oraclejdk11
```

Comment:
You also need to specify `- jdk: oraclejdk11` in the include section :)

Reply:
This diff just looks weird. Take a look at the actual file (I removed the allow_failures section entirely).

Reply:
Aw, my bad I misread the diff